packages/polaris-core/src/config/permissions-configuration.ts

```diff
@@ -0,0 +1,9 @@
+import { PolarisGraphQLContext } from '@enigmatis/polaris-common';
+
+export interface PermissionsConfiguration {
+    customPermissionsFunction?: (
```

Comment:
not sure about the name, cause its only activated if you has requesting system.
maybe systemPermissionsFunction?